### PR TITLE
Jenkins.monthly

### PR DESCRIPTION
**Pull request by Mistreated**
## Basics

These points need to be fulfilled for every PR:

- [ ] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [ ] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy.
- [ ] The PR is rebased with current master.

If you have any troubles fulfilling these criteria, please write
about the trouble as comment in the PR. We will help you.
But we cannot accept PRs that do not fulfill the basics.

## Checklist

Check relevant points but **please do not remove entries**.
For docu fixes, spell checking, and similar none of these points below
need to be checked.

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://master.libelektra.org/doc/CODING.md))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned every code not directly written by me in [THIRD-PARTY-LICENSES](doc/THIRD-PARTY-LICENSES)

## Review

Reviewers will usually check the following:

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

If you are already Elektra developer:

- Add the "work in progress" label if you do not want the PR to be reviewed yet.
- Add the "ready to merge" label **if the basics are fulfilled** and you also
  say that everything is ready to be merged.

### CODE REVIEW

File: scripts/jenkins/Jenkinsfile.monthly [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env groovy
+
+stage('Trigger monthly libelektra/master build') {
+    build job: 'libelektra/master', wait: false
```
Comment (markus2330) [inline scripts/jenkins/Jenkinsfile.monthly]: In the other Jenkinsfile we use two spaces.

Comment (mpranj) [reply scripts/jenkins/Jenkinsfile.monthly]: @Mistreated can you unify the style (use two spaces) and rebase so we can merge this?

**Comment by markus2330** [comment]
@Mistreated some builds are still failing.

**Comment by Mistreated** [comment]
I get the following error on the failed builds:

`brew install python@2; brew link --overwrite python@2
Error: No available formula with the name "python@2" `

I did not see that on the other builds. Is there more info to it?

**Comment by sanssecours** [comment]
> I get the following error on the failed builds:
> 
> `brew install python@2; brew link --overwrite python@2 Error: No available formula with the name "python@2" `
> 
> I did not see that on the other builds. Is there more info to it?

As far as I know, pull request #3351 already fixed this problem. Please just rebase on `master`.

**Comment by mpranj** [comment]
@Mistreated something went wrong with the rebase. It looks like you merged master into your branch. [This](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork) is how you set up an upstream repo url for your fork. Then you can [rebase from upstream master](https://stackoverflow.com/a/7929499).

**Comment by Mistreated** [comment]
`Current branch master is up to date.` is what i get.
@mpranj Should I make a new PR and link it to this one?

**Comment by sanssecours** [comment]
> Should I make a new PR and link it to this one?

I created a cleanly rebased version of the PR [here](https://github.com/ElektraInitiative/libelektra/pull/3368). If you want we can merge that pull request instead of this one, after all build jobs finish successfully.

**Comment by Mistreated** [comment]
@sanssecours Thank you!

**Pull request closed without merging**